lib/poller.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const { EventEmitter } = require('events')
+
+class Poller extends EventEmitter {
```

Comment:
the use of `EventEmitter` here confuses me:
1. some of the event names are pretty long and seem human readable (e.g., `split data updated`) -- is the human readability for some purpose?
2. how does one use this class? can you describe the events you expect an implementer to listen to?  e.g., `const UPDATE = 'update'; const ERROR = 'error'`

Reply:
yeahhh.
for 1) i guess i was really treating `emit` as a logger for my testing. for 2) i'd be thinking the implementer would listen for updates, but hadn't been thinking about it in detail.

i've updated to use `setInterval` which seems simpler for now, though i'm still working out `stop`